### ISSUE TITLE
Speaker notes

### ISSUE THREAD

**Issue by trevorcampbell** [jupyterlab-contrib/rise#47]
Thank you for porting RISE over to JupyterLab!

In the original jupyter notebook RISE, I was able to open a "speaker notes" window that would follow along with the slides presentation, except it would show "notes" cells that are otherwise hidden in the presentation. I've been using this to give myself code skeletons for live coding demos. Because it's in a separate browser window, I can keep the speaker notes privately just on my own screen while the main presentation slides are shown to the audience.

Is there any chance of doing this in JupyterLab RISE?

- JupyterLab RISE version

```
jupyterlab_rise               0.40.0
```

- Other relevant package versions:
```
jupyter_client                8.3.0
jupyter_core                  5.3.1
jupyter-events                0.7.0
jupyter-lsp                   2.2.0
jupyter_server                2.7.1
jupyter-server-mathjax        0.2.6
jupyter_server_terminals      0.4.4
jupyter-telemetry             0.1.0
jupyterhub                    4.0.2
jupyterlab                    4.0.5
jupyterlab-git                0.41.0
jupyterlab-pygments           0.2.2
jupyterlab_rise               0.40.0
jupyterlab_server             2.24.0
jupyterlab-widgets            3.0.8
nbclassic                     1.0.0
nbclient                      0.8.0
nbconvert                     7.7.4
nbdime                        3.2.1
nbformat                      5.9.2
nbgitpuller                   1.2.0
nbgrader                      0.9.0a1
widgetsnbextension            4.0.8
```

**Comment by nthiery** [comment]
Just for information: typing `t` here does open a new window containing speaker notes.
I don't know if it's usable though, kept properly in sync with the window without
speaker notes, etc.

**Comment by trevorcampbell** [comment]
> typing t here does open a new window containing speaker notes.

Huh! When I was trying the new RISE out a few days ago, it definitely did not work. Mind you, I only tried the shortcut key once or twice before going to find in the documentation whether that feature was ported over. Just loaded it up again to try, and *again* it didn't work. But then I mashed `t` many times, and it *eventually* worked.

Not really sure what happened there, but it doesn't sound easily reproducible ;) so we can close this unless someone else finds the same problem. Whatever happened on my machine is most likely some idiosyncracy with keyboard focus or some such...
